environments/workspaces-as-code.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Workspaces As Code"
+description: "Learn how to describe environment configuration as code."
+state: beta
+---
+
+Workspaces as Code (WAC) brings Infrastructure As Code to environment
+configuration. Users can use this feature to manage environment config in the
+same repo as their source code; simply add a `.coder/coder.yaml` to get started.
+
+## Open In Coder Badge
+
+In order to make it easy for your users to take advantage of your WAC config,
+file you can add an "Open in Coder" badge to your repository's `README.md`.
```

Comment:
This should be moved to `Admin` and users should be referred to as developers

Reply:
Yeah as was worried about discoverability, do we just add a reference to it from the index doc?

Reply:
I think that's a great idea